extensions/pom.xml

```diff
@@ -112,6 +112,7 @@
 
         <!-- kogito -->
         <module>kogito</module>
```

Comment:
I there a reason why the module `kogito` is included ? Can we avoid that this extension example project extend the quarkus extension parent pom ? @danielpetisme